seata-spring-autoconfigure/seata-spring-autoconfigure-server/src/main/java/io/seata/spring/boot/autoconfigure/properties/server/store/StoreProperties.java

```diff
@@ -57,7 +57,7 @@ public StoreProperties setPublicKey(String publicKey) {
     @Component
     @ConfigurationProperties(prefix = STORE_SESSION_PREFIX)
     public static class Session {
-        private String mode = "file";
+        private String mode;
 
         public String getMode() {
             return mode;
```

Comment:
Pls print log warning the item has expired when calling store.mode, but the function is compatible with previous and available.